react-ui/src/components/Layouts/InputWorkerTimes/AvailableMeetingHeader.css

```diff
@@ -0,0 +1,23 @@
+.appHeader {
+    height: 40px;
+    background-color: rgb(93, 93, 226);
```

Comment:
Hi Kyle,
Side note, but it may be better to leverage the constants in the sass file (yet to be merged) so that our whole application has a similar colour styling.

Reply:
If I set up my code routing to work like the other links on the page, then the appheader file can be deleted entirely

Reply:
Sure, if it's not needed, then I would remove it so our website has a consistent look and feel. If you do have a specific reason as to why you want to keep it, just make a comment and you can also leave it as well.

Reply:
Code has been removed